compiler/compile/OMRCompilation.cpp

```diff
@@ -2713,3 +2714,17 @@ OMR::Compilation::CompilationPhaseScope::~CompilationPhaseScope()
    {
    _comp->restoreCompilationPhase(_savedPhase);
    }
+
+TR::Region &OMR::Compilation::aliasRegion()
+   {
+   return self()->_aliasRegion;
+   }
+
```

Comment:
This file is changed in a commit under Mem14 already reviewed by @0xdaryl, but @lmaisons is welcome to comment in Mem14